projects/news_feed/utilities.py

```diff
@@ -14,32 +14,117 @@
 import requests
 from bs4 import BeautifulSoup
 
+# printing message for user
+print('Fetching PSL 2020 matches data. Please wait ...')
```

Comment:
not correct place for this print.
it shall be in your `main` function in your other script

Reply:
@M-Shan I had added it to main function earlier. However it was printing just before the actual matches data. So I thought that I should add it here because python is checking my utilities file first before jumping to call functions in main file. consequently I got much better time gap between intermediate print and actual one.
correct me if I am wrong!

Reply:
its ok to move inside function `main` 
also avoid creating global variables in files and try to write prints in functions